playbooks/roles/mattermail/defaults/main.yml

```diff
@@ -0,0 +1,18 @@
+# Connections settings for Mattermost
+MATTERMOST_URL: https://mattermost.example.com
+MATTERMOST_TEAM: opencraft
+MATTERMOST_LOGIN: mattermail
+MATTERMOST_PASSWORD: set-me-please
```

Comment:
This variable isn't actually used, and to be honest a lot of these variable names are pretty ambiguous, e.g. what is the difference between `MATTERMOST_PASS` and `MATTERMOST_PASSWORD`?

Also, some variables, like `MATTERMOST_URL`, would I think basically be redefinitions of the existing `MATTERMOST_HOSTNAME` from the `mattermost` role. You already created a separate role at this point, but since Mattermail is so connected to Mattermost itself, do you think it'd make sense to merge this stuff into the existing `mattermost` role, with a flag that sets whether to install Mattermail or not? It can be brought in at the end of the `mattermost` role's task steps with `import_tasks` according to whether that conditional is set, if you decide to do this.

Reply:
I wanted to make this as independent as possible since Braden mentioned writing a Procmail filter to accomplish this in the future.

Reply:
I haven't looked into Procmail and likely won't have time, but wouldn't utilizing it mean mostly a complete replacement of most of this role anyway?

Reply:
@xitij2000 That's confusing alright. For MATTERMOST_URL both should be the same, so is it safe to just reuse the values from another role?
@UmanShahzad That's right, which is why I thought of keeping a separate role instead of integrating with Mattermost role.

Reply:
I just mean to do that for variable names that convey the same information but need two differnt names for some reason.